CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,120 @@
+# Code of Conduct
+
+([Français](#Code-de-conduite))
+
+Contributors are expected to follow the Code of Conduct, and those working within Government are also expected to follow the Values and Ethics Code for the Public Sector
+
+## Values and Ethics Code for the Public Sector
+
+The [Values and Ethics Code for the Public Sector](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=25049)
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
I suggest ending the pledge at "...harassment-free experience for everyone." 

"Everyone" is everyone - 100% of humans. :) It's not necessary to expand on that. This also keeps the pledge a solid, descriptive but succinct sentence.

Reply:
Thanks; I think keeping this helps in ensuring and promoting a clear shared understanding and awareness on the side of the maintainers/contributors